candevstudio/docs/1_Usage.md

```diff
@@ -0,0 +1,11 @@
+# Usage
```

Comment:
Add references to host install doc (docs/host-configuration/...)
Also add some comments and a reference to official CANdevStudio website (or repo)